game/src/ungap/mod.rs

```diff
@@ -55,12 +56,13 @@ impl ExploreMap {
 
         Box::new(ExploreMap {
             top_panel: Panel::empty(ctx),
-            legend: make_legend(ctx, app, true, false),
+            bottom_right_panel: make_bottom_right_panel(ctx, app, true, false),
```

Comment:
I wanted the zoom controls to sit just above the legend, so I put the legend and the zoom controls into the same panel and made this name more general.